packages/lodestar/src/network/libp2p/rpc.ts

```diff
@@ -152,6 +154,7 @@ export class NetworkRpc extends EventEmitter {
         delete this.requests[id];
         reject(new Error(`request timeout, method ${method}, id ${id}`));
       }, this.requestTimeout);
+      // @ts-ignore
```

Comment:
same, can we include a comment about `response ${id}` events in INetworkRpcEvents?